test/Concurrency/sending_interfacefile_printing.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend %s -typecheck -enable-library-evolution -parse-as-library -emit-module-interface-path - -module-name MyFile -swift-version 6 | %FileCheck -DFILE_PREFIX=MyFile %s
+// RUN: %target-swift-frontend %s -typecheck -enable-library-evolution -parse-as-library -emit-module-interface-path - -module-name MyFile -swift-version 6 -Xllvm --swift-ast-printer-force-printing-types-as-type-reprs | %FileCheck %S/Inputs/sending_interfacefile_printing_repr_filecheck
```

Comment:
I always find tests that have separate inputs like this to be a bit difficult to follow. Since module qualification is not the thing you're testing, maybe the check lines could just be looser?

```
CHECK-NEXT: public func test() -> sending {{MyFile.|}}NonSendableKlass
```

Reply:
I tried that actually. FileCheck regex doesn't support that.

Reply:
```
swift/test/Concurrency/sending_interfacefile_printing.swift:20:48: error: invalid regex: empty (sub)expression
// CHECK-NEXT: public func test() -> sending {{MyFile.|}}NonSendableKlass
                                               ^
```

Reply:
Not a biggie, but I think you could just do it via CHECK prefixes instead to avoid needing a separate input file:
```
// CHECK-NORMAL-NEXT: public func test() -> sending MyFile.NonSendableKlass
// CHECK-PRESERVE-NEXT: public func test() -> NonSendableKlass
```